backend/src/services/auth.rs

```diff
@@ -46,8 +46,8 @@ impl AuthService {
 
         Email {
             sender: (
-                Some("Hack4Krak Authentication".to_string()),
-                "auth@hack4krak.pl".to_string(),
+                Some("Autoryzacja Hack4Krak".to_string()),
+                "autoryzacja@hack4krak.pl".to_string(),
```

Comment:
I think we can leave email as it is. I couldn't find any site that uses non-english in email address